packages/documentsearch/src/interfaces.ts

```diff
@@ -29,6 +29,11 @@ export interface IDisplayState {
    */
   useRegex: boolean;
 
+  /**
+   * Should the search be within single cell?
+   */
+  searchCell: boolean;
```

Comment:
How about something like
Select if the search happens on the entire document or just a single cell
`Should the search be within single cell?`

Also `searchCell` could be `searchSingleCell` or `searchActiveCell` to make it more descriptiive?

Thoughts @blink1073 ?

Reply:
Thanks for the suggestion, will make this change along with others